resource/resource.go

```diff
@@ -19,6 +19,8 @@
 // and stylesheets.
 package resource
 
+//go:generate tsc -p ui/ts/
```

Comment:
For now, I think this makes sense.  We can always convert it to a make js command later on.